test/unit/notifications.server.test.ts

```diff
@@ -0,0 +1,44 @@
+import invariant from 'tiny-invariant';
+import sendNotification from '../../app/lib/notifications.server';
+
+const { NOTIFICATIONS_EMAIL_USER } = process.env;
+
+describe('Notification server lib function test', () => {
+  const recipient = 'dev@starchart.invalid';
+  const subject = 'Sample subject';
+  const text = 'Sample text';
+
+  test('sending and receiving email notification using sendNotification()', async () => {
+    const sentMailResponse = await sendNotification(recipient, subject, text);
+
+    // prevent typescript error: messageId is possibly undefined
+    invariant(sentMailResponse?.messageId, 'Expected sentMailResponse.messageId to be defined');
+    let messageId = sentMailResponse.messageId;
+
+    expect(sentMailResponse).toEqual(
+      expect.objectContaining({
+        accepted: expect.arrayContaining([recipient]),
+      })
+    );
+
+    const mailhogResponse = await fetch(
+      `http://localhost:8025/api/v2/search?kind=to&query=${recipient}`
+    );
+    const responseToJSON = await mailhogResponse.json();
+    const mostRecentMessage = responseToJSON.items;
+    expect(mostRecentMessage).toEqual(
```

Comment:
I think you can remove this variable entirely

Reply:
```suggestion
    expect(responseToJSON.items).toEqual(
```

Reply:
Good catch. Thanks.